content/intro-to-storybook/react/nl/deploy.md

```diff
@@ -1,101 +1,142 @@
 
 ## Exporteren als een statische app
 
-Om Storybook te deployen, moeten we het eerst exporteren als een statische web-app. Deze functionaliteit is al ingebouwd in Storybook, we moeten het alleen activeren door een script toe te voegen aan `package.json`.
-
-```javascript
-// package.json
+Om Storybook te deployen, moeten we het eerst exporteren als een statische web-app. Deze functionaliteit is ingebouwd in Storybook en alreeds geconfigureerd.
 
-{
-  "scripts": {
-    "build-storybook": "build-storybook -c .storybook"
-  }
-}
-```
+`yarn build-storybook` uitvoeren, zal een static Storybook in de `storybook-static` folder opleveren, dewelke dan kan gedeployed worden naar een static site hosting service naar keuze.
 
-Wanneer je nu Storybook buildt via `npm run build-storybook`, zal het een statisch Storybook outputten in de folder `storybook-static`.
+## Storybook publiceren
 
-## Continuous deploy
+Deze tutorial gebruikt <a href="https://www.chromatic.com/">Chromatic</a>, een gratis pblishing service gemaakt door de mensen die Storybook onderhouden. Het laat ons toe om Storybook veilig in de cloud te deployen.
 
-We willen de nieuwste versie van componenten delen wanneer we code pushen. Om dit te doen moeten we Storybook continu deployen. We zullen op GitHub en Netlify vertrouwen om onze statische site te deployen. We gebruiken het gratis Netlify-abonnement.
+### Een GitHub repository opzetten
 
-### GitHub
+Voor we beginnen, moet onze lokale code synchroniseren met een remote versiecontrolesysteem. Toen we ons project initialiseerden in het [Get started hoofdstuk](/react/nl/get-started), had Create React App (CRA) alreeds een lokale repository voor ons gemaakt. Op dit moment is het oké om onze bestanden toe te voegen aan de eerste commit.  
 
-Eerst wil je Git instellen voor je project in de lokale folder. Als je hebt gevolgd met het vorige hoofdstuk over testing, ga dan naar het opzetten van een repository op GitHub.
+Voor de volgende commando's uit om de veranderingen die we tot dusver hebben gemaakt toe te voegen en te committen.
 
 ```bash
-$ git init
+$ git add .
 ```
 
-Voeg vervolgens bestanden toe aan de eerste commit.
+Ga naar GitHub om [hier](https://github.com/new) een nieuwe repository voor ons project te creëren. Noem de repo “taskbox”, hetzelfde als ons lokaal project.
+
+![GitHub instelling](/intro-to-storybook/github-create-taskbox.png)
+
+Kopieer in de nieuwe repo-setup de oorspronkelijke URL van de repo en voeg deze toe aan je git-project met dit commando:
 
 ```bash
-$ git add .
+$ git remote add origin https://github.com/<your username>/taskbox.git
 ```
 
-Commit nu de bestanden.
+Tenslotte, push de lokale repo naar de remote repo op GitHub met het volgende:
 
 ```bash
-$ git commit -m "taskbox UI"
+$ git push -u origin master
```

Comment:
```suggestion
$ git push -u origin head
```

In case they're using something other than `master` (e.g. `main`), this will work either way.